Admin tools

GoralTomaszGorski opened the pull request:
- added additional calendar therm function
- edit/delete function prepared

GoralTomaszGorski committed:
templates reduced, email for patients added
Improved style and view,

GoralTomaszGorski committed:
templates reduced, email for patients added
Improved style and view, Services

GoralTomaszGorski committed:
templates reduced, email for patients added
Improved style and view, Services

GoralTomaszGorski committed:
- added additional calendar therm function
- edit/delete function prepared

GoralTomaszGorski committed:
user automatically email adding
and setNick during addPatient function
templates reduced, email for patients added
Improved style and view, Services

GoralTomaszGorski committed:
user automatically email adding
and setNick during addPatient function
templates reduced, email for patients added
Improved style and view, Services
DeletePatientException one more fixed.

GoralTomaszGorski committed:
- user: automatically email adding
and setNick automatically during addPatient function
- templates reduced
- Improved style and view, Services
- DeletePatientException one more fixed different way.

GoralTomaszGorski committed:
- added additional calendar therm function
- edit/delete function prepared

GoralTomaszGorski committed:
- added additional calendar therm function
- edit/delete function prepared
- delete calendar therm exception

GoralTomaszGorski committed:
- added additional calendar therm function
- edit/delete function prepared
- delete calendar therm exception

GoralTomaszGorski committed:
- edit/delete therm function prepared
- added additional calendar therm function
- edit/delete calendar function prepared
- delete calendar therm exception

GoralTomaszGorski committed:
- edit/delete therm function prepared
- added additional calendar therm function
- edit/delete calendar function prepared
- delete calendar therm exception

gorskimarcin96 reviewed (approved):
its nice. we have one typo "umuwiony"

GoralTomaszGorski committed:
- edit/delete therm function prepared
- added additional calendar therm function
- edit/delete calendar function prepared
- delete calendar therm exception

GoralTomaszGorski committed:
- edit/delete therm function prepared
- added additional calendar therm function
- edit/delete calendar function prepared
- delete calendar therm exception

GoralTomaszGorski committed:
- meeting searching function
- edit/delete therm function prepared
- added additional calendar therm function
- edit/delete calendar function prepared
- delete calendar therm exception

GoralTomaszGorski committed:
- meeting searching function
- edit/delete therm function prepared
- added additional calendar therm function
- edit/delete calendar function prepared
- delete calendar therm exception

GoralTomaszGorski committed:
- meeting searching function
- edit/delete therm function prepared
- added additional calendar therm function
- edit/delete calendar function prepared
- delete calendar therm exception

GoralTomaszGorski committed:
- meeting searching function
- edit/delete therm function prepared
- added additional calendar therm function
- edit/delete calendar function prepared
- delete calendar therm exception

GoralTomaszGorski committed:
- set free and true calendar therm fixed
- meeting searching function
- edit/delete therm function prepared
- added additional calendar therm function
- edit/delete calendar function prepared
- delete calendar therm exception

GoralTomaszGorski committed:
- set free and true calendar therm fixed
- meeting searching function
- edit/delete therm function prepared
- added additional calendar therm function
- edit/delete calendar function prepared
- delete calendar therm exception
